Specification.md

```diff
@@ -101,7 +103,9 @@ When a node is in _Steady State_:
 - Incoming Sync Interest is up-to-date or newer.
   - No indication of inconsistencies. The scheduled Sync Interest can be delayed.
   - Eventually update the local state and reset Sync Interest Timer to 30 seconds (±10% uniform)
+  - If the incoming Sync Interest has newer entries, store the current timestamp as the last update time for each updated node.
 - Incoming Sync Interest is outdated: Node moves to _Suppression State_
+  - If every outdated node was updated within the last suppression period, drop the Sync Interest and move to _Steady State_
```

Comment:
What does "outdated node" mean?
Section 3 only defined the concept of *outdated* between two StateVectors, but not nodes.